styles/diamond-ui.css

```diff
@@ -1 +1,3 @@
-@import url('./variables.css');
+@import url('../node_modules/modern-normalize/modern-normalize.css');
+@import url('./tokens.css');
+@import url('./base.css');
```

Comment:
This file is in bootstrap CSS framework territory because dropping it in will apply a bunch of prescriptive styles globally, rather than limiting scope to components.

But I suppose we need to work like this to avoid redeclaring stuff in each component?

Reply:
Yeah, I suppose it's whether we see diamond as components only or base styles as well.

Given the principle of styling the html, then adding components to enhance that html it feels like we should have some HTML styles.

I've separated out base and tokens, because you _could_ just use the tokens and the components if you wanted